[mandb] Unable to build from source

```
man.c:338:2: error: 'html_pager' undeclared (first use in this function); did you mean 'init_html_pager'?
  338 |  html_pager = getenv ("BROWSER");
      |  ^~~~~~~~~~
      |  init_html_pager
man.c:338:2: note: each undeclared identifier is reported only once for each function it appears in
man.c: In function 'make_roff_command':
man.c:1308:26: error: 'TROFF' undeclared (first use in this function)
 1308 |       (get_def ("troff", TROFF));
      |                          ^~~~~
man.c:1312:26: error: 'NROFF' undeclared (first use in this function)
 1312 |       (get_def ("nroff", NROFF));
      |                          ^~~~~
man.c: In function 'format_display':
man.c:1922:20: warning: unused parameter 'man_file' [-Wunused-parameter]
 1922 |        const char *man_file)
      |        ~~~~~~~~~~~~^~~~~~~~
  CC       man-recode.o
At top level:
man.c:336:13: warning: 'init_html_pager' defined but not used [-Wunused-function]
  336 | static void init_html_pager (void)
      |             ^~~~~~~~~~~~~~~
man.c:262: warning: macro "ONLY_TROFF_IS_GROFF" is not used [-Wunused-macros]
  262 | #  define ONLY_TROFF_IS_GROFF OPTION_HIDDEN
      | 
make[3]: *** [Makefile:2039: man.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/usr/local/tmp/crew/man-db-2.9.4.tar.xz.dir/man-db-2.9.4/src'
make[2]: *** [Makefile:2075: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/man-db-2.9.4.tar.xz.dir/man-db-2.9.4/src'
make[1]: *** [Makefile:1633: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/man-db-2.9.4.tar.xz.dir/man-db-2.9.4'
make: *** [Makefile:1561: all] Error 2
mandb failed to install: `make V=0 -j2` exited with 2
```

--- a/install.sh
+++ b/install.sh
@@ -17,7 +17,7 @@ CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 CURL="${CURL:-curl}"
 
 EARLY_PACKAGES="gcc10 llvm brotli c_ares libcyrussasl libiconv libidn2 \
-libmetalink libnghttp2 libpsl libssh2 libtirpc libunistring openldap \
+libmetalink libnghttp2 libpsl libssh2 libtirpc libunistring lz4 openldap \
 rtmpdump zstd ncurses ca_certificates ruby libffi openssl nettle krb5 \
 p11kit libtasn1 gnutls curl git icu4c libedit"
 

--- a/packages/mandb.rb
+++ b/packages/mandb.rb
@@ -23,6 +23,7 @@ class Mandb < Package
   })
 
   depends_on 'gdbm'
+  depends_on 'groff'
   depends_on 'libpipeline'
   depends_on 'libseccomp'
 

--- a/tools/core_packages.txt
+++ b/tools/core_packages.txt
@@ -24,6 +24,7 @@ git
 glibc
 gmp
 gnutls
+groff
 icu4c
 intltool
 isl
@@ -81,6 +82,7 @@ sed
 slang
 sqlite
 trousers
+uchardet
 unzip
 util_macros
 wget

